consensus/src/epoch_manager.rs

```diff
@@ -399,10 +399,9 @@ impl EpochManager {
                 .map_err(|err| {
                     error!(
                         SecurityEvent::ConsensusInvalidMessage,
-                        StructuredLogEntry::default()
-                            .data("from_peer", &peer_id)
-                            .data_display("error", &err)
-                            .data("event", &unverified_event)
+                        remote_peer = peer_id,
+                        error = err.to_string(),
```

Comment:
I suppose I could have done this before :D We could add some support for the macros to select if you want to do debug, serde, or display based on a sigil

Reply:
Yeah, I think we can do that in the future, like tracing, I've had to handle this in a couple places.  But, right now just been churning through and getting it done

Reply:
100% agreed